types/util.promisify/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for util.promisify 1.0
+// Project: https://github.com/ljharb/util.promisify#readme
+// Definitions by: Adam Voss <https://github.com/adamvoss>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ Note that ES6 modules cannot directly export callable functions.
+ *~ This file should be imported using the CommonJS-style:
```

Comment:
You're welcome, but really it is `dts-gen` that gets the credit :smiley:.  I just didn't delete the comment because it seemed relevant.